### PR TITLE
feat: add navigation

### PR DESCRIPTION
**Pull request by Misiorny**
feat: add navigation 
fix: path home

### CODE REVIEW

File: src/components/Navigation/NavItem.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Link as Route} from 'react-router-dom';
+import {Link} from '@chakra-ui/react'
+
+function NavItem(state: { to: string; label: string }) {
+    return (
+        <li >
```
Comment (dblach-intive) [inline src/components/Navigation/NavItem.tsx:7]: Why you dont use ListItem from chakra ?

File: src/components/Navigation/dataLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const dataLinks = [
+    { id:1, to: '/dashboard', label: 'Home'},
```
Comment (dblach-intive) [inline src/components/Navigation/dataLinks.tsx:2]: you can use defined links from routes.ts

File: src/components/Navigation/dataLinks.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const dataLinks = [
```
Comment (dblach-intive) [inline src/components/Navigation/dataLinks.tsx:1]: dataLinks array should be in different file for example in exsisitng routes.ts, and change the name for example menuLinks

File: src/components/Navigation/index.tsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import Navbar from "./Navbar";
```
Comment (dblach-intive) [inline src/components/Navigation/index.tsx:1]: you can put all code in this place so index.tsx should contain all from Navbar.tsx

File: src/components/Navigation/NavItemsList.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import NavItem from './NavItem';
+import {Menu} from '@chakra-ui/react'
+import dataLinks from "./dataLinks";
+
+function NavItemList() {
+    return (
+        <Menu >
```
Comment (dblach-intive) [inline src/components/Navigation/NavItemsList.tsx:8]: remove white space at the end

File: src/components/Navigation/NavItemsList.tsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+            {
+                dataLinks.map((e) =>
+                    <NavItem key={e.id} to={e.to} label={e.label} />)
+            }
+        </Menu >
```
Comment (dblach-intive) [inline src/components/Navigation/NavItemsList.tsx:13]: remove white space at the end

**Review by dblach-intive — CHANGES_REQUESTED** [review]

**Comment by dblach-intive** [comment]
all changes form Pr should be in separate commit

File: src/components/Navigation/NavItem.tsx (hunk 1 above)
Comment (Misiorny) [reply src/components/Navigation/NavItem.tsx:7]: ListItem or List? With ListItem doesn't work properly.

File: src/types/routes.ts [hunk 7]
```diff
@@ -1,4 +1,15 @@
 export enum ROUTES {
     AUTH = '/',
     DASHBOARD = '/dashboard'
-}
\ No newline at end of file
+}
+
+const menuLinks = [
+    { id:1, to: '/dashboard', label: 'Home'},
```
Comment (dblach-intive) [inline src/types/routes.ts:7]: You can use ROUTES to define links, add missed links inro Routes too

File: src/components/Navigation/NavItem.tsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Link as Route} from 'react-router-dom';
+import {Link, List} from '@chakra-ui/react'
+
+function NavItem(state:
```
Comment (dblach-intive) [inline src/components/Navigation/NavItem.tsx:5]: Add type for NavItem, i saw it in Patryk review ready to paste

File: src/components/Menu/Menu.tsx [hunk 9]
```diff
@@ -1,10 +1,12 @@
 import React from "react";
```
Comment (dblach-intive) [inline src/components/Menu/Menu.tsx:1]: This file should be index.tsx not Menu.tsx

**Review by dblach-intive — CHANGES_REQUESTED** [review]

File: src/components/Navigation/NavItem.tsx [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+                     {
+                         to: string;
+                         label: string
+                     }) {
+    return (
+        <List >
```
Comment (dblach-intive) [inline src/components/Navigation/NavItem.tsx:11]: This is wrong component it creates UL on every element why dont you use these elements ?
Menu,
  MenuButton,
  MenuList,
  MenuItem,
  MenuItemOption,
  MenuGroup,
  MenuOptionGroup,
  MenuDivider,

**Review by dblach-intive — CHANGES_REQUESTED** [review]

File: src/components/Menu/Menu.tsx (hunk 9 above)
Comment (Misiorny) [reply src/components/Menu/Menu.tsx:1]: Sorry but into Menu.tsx ( not mine component but layout component) I put only Navbar component for install navigation in proper place.

File: src/components/Navigation/NavItem.tsx (hunk 10 above)
Comment (Misiorny) [reply src/components/Navigation/NavItem.tsx:11]: Now structure will be Menu->MenuItem->Link

File: src/components/Navigation/dataLinks.tsx (hunk 2 above)
Comment (dblach-intive) [reply src/components/Navigation/dataLinks.tsx]: yes

**Pull request merged**